package.json

```diff
@@ -250,8 +250,8 @@
 		"test-unit:debug": "wp-scripts --inspect-brk test-unit-js --runInBand --no-cache --verbose --config test/unit/jest.config.js ",
 		"test-unit:update": "npm run test-unit -- --updateSnapshot",
 		"test-unit:watch": "npm run test-unit -- --watch",
-		"test-unit-php": "wp-env run phpunit 'phpunit -c /var/www/html/wp-content/plugins/gutenberg/phpunit.xml.dist --verbose'",
-		"test-unit-php-multisite": "wp-env run phpunit 'WP_MULTISITE=1 phpunit -c /var/www/html/wp-content/plugins/gutenberg/phpunit/multisite.xml --verbose'",
+		"test-unit-php": "npm run wp-env run phpunit 'phpunit -c /var/www/html/wp-content/plugins/gutenberg/phpunit.xml.dist --verbose'",
```

Comment:
Doesn't `npm` look for a local module before falling back to the global?

`"env": "wp-scripts env"`

Also works like this.

Reply:
I think @epiqueras is right. I just tried it without the `npm run` and it still works. So it seems like it's simply a case of not having run `npm run wp-env start` first before trying to run the phpunit tests. Updating the documentation helps for sure, but perhaps trying to conditionally provision WP when calling `run phpunit` before `start` would be a good option too.

Reply:
I'm not very familiar with how npm resolution works, but this also makes it more consistent with other places we call wp-env in package.json

Reply:
I switched it back to wp-env run by itself and made the other instances consistent with this :)